antlr-kotlin-gradle-plugin/src/main/java/com/strumenta/antlrkotlin/gradleplugin/AntlrKotlinPlugin.java

```diff
@@ -42,7 +42,7 @@
 public class AntlrKotlinPlugin implements Plugin<Project> {
     private static final Logger LOGGER = LoggerFactory.getLogger(AntlrKotlinPlugin.class);
 
-    private static final String ANTLR_CONFIGURATION_NAME = "antlrkotlin";
+    private static final String ANTLR_CONFIGURATION_NAME = "antlr-kotlin";
```

Comment:
Is this necessary?

Reply:
Not at all, but nicer. Does this affect anything outside?

Reply:
Ok, makes sense